packages/gafl-webapp-service/src/services/analytics/analytics.js

```diff
@@ -0,0 +1,15 @@
+
+export const getTrackingProductDetailsFromTransaction = transaction =>
+  transaction.permissions.map(permission => ({
+    id: permission.permit.description,
+    name: `${permission.permit.permitSubtype.label} - ${permission.permit.numberOfRods} rod(s) licence`,
+    brand: permission.permit.permitType.label,
+    category: [
+      permission.permit.permitSubtype.label,
+      `${permission.permit.numberOfRods} rod(s)`,
+      permission.permit.concessions.length ? permission.permit.concessions.join(',') : 'Full'
```

Comment:
The concessions array is an array of object, not string.  We'll need to do a join on each concession object's name property:
```
[
  {
    "id": "d1ece997-ef65-e611-80dc-c4346bad4004",
    "name": "Disabled"
  }
]